aspnetBlazor/Dockerfile

```diff
@@ -18,7 +18,7 @@ ARG APP_ROOT=
 
 
 # BUILD ------------------------------------------------------------------------
-FROM mcr.microsoft.com/dotnet/sdk:8.0 AS Build
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
```

Comment:
as you changed here the `Build` to `build` the `COPY` in line number 55 will fail.
Please check if we are referencing the stage name because we will need to update them all.